final_prep/annotations.rb

```diff
@@ -4,10 +4,13 @@
 
 # Build a Bear
 
-def build_a_bear(name, age, fur, clothes, special_power)
+def build_a_bear(name, age, fur, clothes, special_power) #create function with 5 arguments
```

Comment:
IMO, It's a littler easier to read through annotations when they're placed directly above the code they're written for.